autoload.php

```diff
@@ -42,6 +42,16 @@ function codecept_debug($data)
     }
 }
 
+if (!function_exists('codecept_debug_mode')) {
```

Comment:
Wrong function name used in this condition.

Reply:
Ok. I'll create a new branch based on 3.0 and place another PR (fixing this function name issue).

Reply:
New PR created: https://github.com/Codeception/Codeception/pull/5502